alert-common/src/main/java/com/synopsys/integration/alert/common/rest/model/AlertPagedDetails.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * alert-common
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.alert.common.rest.model;
+
+import java.util.List;
+
+import com.synopsys.integration.alert.common.rest.api.ReadPageController;
+import com.synopsys.integration.util.Stringable;
+
+public class AlertPagedDetails<M extends Stringable> {
```

Comment:
I would honestly drop the `Stringable`. It's nice to have the constraint, but I think we want this class to be as flexible as possible.